### ISSUE TITLE
Build fails on Xcode. Unity 5.2.3 plugin version 9.27 and Xcode 6.4

### ISSUE THREAD

**Issue by Hoffa25** [playgameservices/play-games-plugin-for-unity#834]
I have been stuck on this for more than 10 hours now....:(. Im new to this cocoapods stuff so maybe its just something simple...?

First the build failed because the GooglePlus.framework was missing. Perhaps there is something wrong with my cocoapod setup? Should they really be red? 
![croppad](https://cloud.githubusercontent.com/assets/10234700/11533153/8720af70-9908-11e5-801f-50cc08e00408.png)

I then imported the GooglePlus.framework manually and then the build went all the way to the "Linking" part. There I get 200+ of Apple Mach-O Linker errors. I have Scripted Backend set to "LL2CPP".
![asdasdasd](https://cloud.githubusercontent.com/assets/10234700/11535878/2b8f386c-9916-11e5-9542-f672e35e6640.png)

**Comment by claywilkinson** [comment]
They should not be red.  The linker errors are clearly unity libraries not being included.  I have not tried 5.2.3, but I have tried 5.2.0 without issue.  One thing you might want to check is that the player settings in Unity are targeting the correct device type (device or simulator), and the architecture is correct (I usually build universal).

Where there any warnings or errors if you run `pod update` in the xcode project directory?

**Comment by Hoffa25** [comment]
Thanks for super quick answer yet again:)! I have device type: Ipad + Iphone and architecture: Universal. No warnings or errors when I run "pod update"... So I tried to downgrade to Unity 5.2.0 and unfortunately I get the same errors when i build in xcode... however one thing is different: "The 3 red lines that shouldnt be red are now just 2, one has become black". The GooglePlus/GooglePlus.h file is still missing and causing build fail (I fix that if I manually import the GooglePlus.framework)... 
![googleplus](https://cloud.githubusercontent.com/assets/10234700/11539980/5e80ad64-992a-11e5-8536-6a76d4703d26.png)


Is the podfile text really supposed to look like that? If im not incorrect I think I saw on some forum that they had two lines. Not just "pod GooglePlayGames" but also "pod GooglePlus/SignIn" or something...
![wasred](https://cloud.githubusercontent.com/assets/10234700/11539987/692596bc-992a-11e5-9f81-4ea3122d9716.png)

**Comment by Hoffa25** [comment]
Actually there were other things that happened when i built with the 5.2.0 version... when i manually fixed the googleplus.framwork and built again I only got 5 linker errors instead of 200+...

**Comment by claywilkinson** [comment]
Those methods are defined in Assets/Plugins/iOS/GPGSAppController.mm.  Unity should have copied them over when creating the xcode project.  I would confirm you have the correct source files in Unity, then confirm they are copied by Unity correctly into the xcode project.

**Comment by Hoffa25** [comment]
Wohoo!! Build succes:). I replaced the GPGSAppController.mm and now it works! I should say that have also updated from OSX Yosemite and from xcode 6.4 to 7.1... but replacing GPGSAppController.mm was the latest thing I did and then it worked! I didnt even  need to manually import the googleplus.workframe. This isnt the first time replacing corrupted plugin files is the solution when there is something funky going on. I will keep that in mind from now on. Thanks alot again Clay! Now I can go to sleep :)
